### PR TITLE
fix: long filter was missing. 

### LINKED ISSUE #1636: Long filter is not yet implemented
**What version of EMX2 are you using (see footer)**
8.117.0

**Describe the bug**
Setting a filter on a long column throws an error and doesn't work

**To Reproduce**
1. Go to 'https://emx2.test.molgenis.org'
2. Click on the pet store
3. Click on the order table
4. Select filters and turn on the quantity filter
5. It says ERROR instead of having an input field

**Expected behavior**
Instead of error it shows an input field for the filter

### PR DESCRIPTION
**Pull request by mswertz**
closes #1636

### CODE REVIEW

File: apps/molgenis-components/src/components/forms/InputRangeLong.vue [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    </InputGroup>
+  </FormGroup>
+</template>
+
+<script>
+import BaseInput from "./baseInputs/BaseInput.vue";
+import BaseInputLong from "./baseInputs/BaseInputLong.vue";
+import FormGroup from "./FormGroup.vue";
+import InputGroup from "./InputGroup.vue";
+import {getBigIntError} from '../utils';
```
Comment (chinook25) [inline apps/molgenis-components/src/components/forms/InputRangeLong.vue:38]: I've noticed that in a lot of these files, the layout is not consistent with our settings. Maybe your edit uses slightly different settings for Vue3? Would be nice if it'd stay the same.

**Review by chinook25 — CHANGES_REQUESTED** [review]
It works, but still throws an not implemented error in TableExplorer.vue (line 458)

File: apps/molgenis-components/src/components/forms/InputRangeLong.vue (hunk 1 above)
Comment (mswertz) [reply apps/molgenis-components/src/components/forms/InputRangeLong.vue:38]: Sorry about the formatting. Meanwhile hopefully fixed the TableExplorer error, thx

**Review by chinook25 — CHANGES_REQUESTED** [review]
Looks good, but I found another bug, having to do with the url encoding of the long filter. 
s2r: go to the pet store, order table. Add quantity filter. Add some values e.g. 5 to 10. Refresh the page. Refreshing breaks the filter, it gets even worse if you add multiple long filters.

**Review by chinook25 — APPROVED** [review]

**Review by chinook25 — CHANGES_REQUESTED** [review]
Adding and then removing values from a long filter leaves an artefact in the url e.g.
http://localhost:8080/pet%20store/tables/#/Order?_filter=quantity&quantity=..
The last part should be removed, as it causes an error on reload.

**Review by chinook25 — APPROVED** [review]

**Pull request merged**